### ISSUE TITLE
Automatically clear  cell outputs when Shift+Enter is pressed

### ISSUE THREAD

**Issue by CavalloneChen** [microsoft/vscode-jupyter#5186]
In jupyter-lab, this is the default behavior. 
This behavior makes sense since Shift+Enter brings focus to the next cell below. When long cell output is kept (current behavior), this leads to a big jump of the viewpoint. And I often found myself in a situation where I have to scroll to find my current location.

**Comment by greazer** [comment]
Current behavior of extension (insiders) seems identical now to Jupyter classic scrolls when shift-enter is pressed. However, there is still a difference in behavior that I think could account for what @CavalloneChen is describing.

In this gif, notice that Jupyter classic prioritizes on keeping the user looking at the same location in the notebook. VS Code is optimizing for seeing the latest output. 
![ScrollBehavior](https://user-images.githubusercontent.com/11685408/131591107-4c8c72d6-ca9a-422f-9e3c-334f4888bfc2.gif)

This almost feels like it could be a setting as either behavior is reasonable.

**Comment by benlindsay** [comment]
> This almost feels like it could be a setting as either behavior is reasonable.

Agreed that this could be a setting. I think defaulting to the behavior seen in Jupyter Lab would be best though, meaning prioritizing not scrolling
